### ISSUE TITLE
Frappedesk Installation Error (Error in query)

### ISSUE THREAD

**Issue by HUMENTH** [frappe/desk#604]
WSL2 UBUNTU 22.04
bench 5.13.1
```
erpnext@HUMENTH:~/frappe-bench$ bench --site frappedesk.test install-app frappedesk

Installing frappedesk...
Updating DocTypes for frappedesk    : [========================================] 100%
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 109, in <module>
    main()
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    click.Group(commands=commands)(prog_name="bench")
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/commands/__init__.py", line 29, in _func
    ret = f(frappe._dict(ctx.obj), *args, **kwargs)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/commands/site.py", line 413, in install_app
    _install_app(app, verbose=context.verbose, force=force)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/installer.py", line 299, in install_app
    frappe.get_attr(after_install)()
  File "/home/erpnext/frappe-bench/apps/frappedesk/frappedesk/setup/install.py", line 14, in after_install
    add_default_agent_groups()
  File "/home/erpnext/frappe-bench/apps/frappedesk/frappedesk/setup/install.py", line 200, in add_default_agent_groups
    if not frappe.db.exists("Agent Groups", agent_group):
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 1075, in exists
    return self.get_value(dt, dn, ignore=True, cache=cache)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 468, in get_value
    result = self.get_values(
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 566, in get_values
    out = self._get_values_from_table(
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 801, in _get_values_from_table
    r = self.sql(query, as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 227, in sql
    traceback.print_stack()
Error in query:
("SELECT `name` FROM `tabAgent Groups` WHERE `name`='Billing' ORDER BY `modified` DESC LIMIT 1", None)
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 109, in <module>
    main()
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    click.Group(commands=commands)(prog_name="bench")
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/erpnext/frappe-bench/env/lib/python3.10/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/commands/__init__.py", line 29, in _func
    ret = f(frappe._dict(ctx.obj), *args, **kwargs)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/commands/site.py", line 413, in install_app
    _install_app(app, verbose=context.verbose, force=force)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/installer.py", line 299, in install_app
    frappe.get_attr(after_install)()
  File "/home/erpnext/frappe-bench/apps/frappedesk/frappedesk/setup/install.py", line 14, in after_install
    add_default_agent_groups()
  File "/home/erpnext/frappe-bench/apps/frappedesk/frappedesk/setup/install.py", line 200, in add_default_agent_groups
    if not frappe.db.exists("Agent Groups", agent_group):
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 1075, in exists
    return self.get_value(dt, dn, ignore=True, cache=cache)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 468, in get_value
    result = self.get_values(
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 566, in get_values
    out = self._get_values_from_table(
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 801, in _get_values_from_table
    r = self.sql(query, as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/home/erpnext/frappe-bench/apps/frappe/frappe/database/database.py", line 227, in sql
    traceback.print_stack()
Error in query:
("SELECT `name` FROM `tabAgent Groups` WHERE `name`='Product Experts' ORDER BY `modified` DESC LIMIT 1", None)
```

**Comment by jahnaviraj** [comment]
Having the same issue 
Ubuntu 22.04 
frappe 14.x.x-develop
frappedesk 0.2.0

**Comment by kamaljohnson** [comment]
Should be resolved in #607
